libbeat/outputs/console/docs/console.asciidoc

```diff
@@ -7,6 +7,8 @@
 
 The Console output writes events in JSON format to stdout.
```

Comment:
One can configure a custom formatter to print messages with a custom layout. JSON is just the default formatter (see codec setting).